checkov/terraform/checks/resource/gcp/GoogleComputeSerialPorts.py

```diff
@@ -12,10 +12,16 @@ def __init__(self):
 
     def scan_resource_conf(self, conf):
```

Comment:
`BaseResourceNegativeValueCheck` defines how resource is scanned via `scan_resource_conf`
That means that its implementing checks shouldn't override it